src/Microsoft.TestPlatform.CrossPlatEngine/EventHandlers/TestRequestHandler.cs

```diff
@@ -22,24 +22,25 @@ namespace Microsoft.VisualStudio.TestPlatform.CommunicationUtilities
 
     public class TestRequestHandler : ITestRequestHandler
     {
+        private const int MinimumProtocolVersionWithDebugSupport = 3;
+
+        private int protocolVersion = 1;
+        private int highestSupportedVersion = 3;
```

Comment:
>highestSupportedVersion [](start = 20, length = 23)

This seems to be defined in multiple places too.

Reply:
Good point. Works fine for cases where we have same `highestSupportedVersion` for both `vstest.console` and `testhost` processes, however it could lead to compat issues in cases where we have an old `vstest.console` coupled with a new `testhost` or the other way around.

To conclude, this is more like a specific highest supported version per component than a constant that can be pulled from, say, `*.ObjectModel.dll`.